ocs_ci/deployment/deployment.py

```diff
@@ -611,6 +611,15 @@ def do_deploy_cnv(self):
         ):
             CNVInstaller().deploy_cnv()
 
+    def do_deploy_metallb(self):
+        """
+        Deploy MetalLB
+        """
+        if config.DEPLOYMENT.get("metallb_operator"):
+            from ocs_ci.deployment.metallb import MetalLBInstaller
```

Comment:
can we move it to import section or if there is any circular dependency issue, please add a comment

Reply:
I'll check that, I've placed it there similarly as it is in the following method `do_deploy_hosted_clusters`,  but I'll check if there actually is some circular dependency issue and update accordingly.

Reply:
Fixed by [33b3d46](https://github.com/red-hat-storage/ocs-ci/pull/10836/commits/33b3d4647eebb8d73d04f25b83c1a6b76c9a6b81)